packages/core/src/modules/ledger/IndyPool.ts

```diff
@@ -31,6 +33,8 @@ export class IndyPool {
   private _poolHandle?: number
   private poolConnected?: Promise<number>
   public authorAgreement?: AuthorAgreement | null
+  // aka the qualified identifier
+  public didIndyNamespace: string
```

Comment:
We can make this a getter (like we did for the id property). This way we don't have to create another property